modules/custom/az_global_footer/az_global_footer.install

```diff
@@ -0,0 +1,37 @@
+<?php
+
+/**
+ * @file
+ * Contains az_global_footer.install.
+ */
+
+use Drupal\migrate\MigrateExecutable;
+use Drupal\migrate\MigrateMessage;
+
+/**
+ * Implements hook_install().
+ */
+function az_global_footer_install() {
+  $tag = 'Quickstart Global Footer';
+
+  // Run the migrations that are tagged as global footer.
+  $migrations = \Drupal::service('plugin.manager.migration')->createInstancesByTag($tag);
+  foreach ($migrations as $migration) {
+    $executable = new MigrateExecutable($migration, new MigrateMessage());
+    $executable->import();
+  }
+}
+
+/**
+ * Implements hook_uninstall().
+ */
+function az_global_footer_uninstall() {
+  $tag = 'Quickstart Global Footer';
```

Comment:
If we also want to disable the theme setting when this module is uninstalled, this is the code we'd need to add for that.

```suggestion
  // Disable clean wrapper for footer_sub region.
  $config = \Drupal::service('config.factory')->getEditable('az_barrio.settings');
  $config
    ->set('bootstrap_barrio_region_clean_footer_sub', FALSE)
    ->save(TRUE);

  $tag = 'Quickstart Global Footer';
```